Docs/source/running_cpp/parameters.rst

```diff
@@ -67,6 +67,13 @@ Overall simulation parameters
     electromagnetic effects (e.g. propagation of radiation, lasers, etc.)
     are not captured.
 
+Setting up in situ visualization
+-------------------------
+
+* ``insitu.int`` (`integer`; 0 by default)
+Turns in situ processing on or off and controls how often data is processed.
```

Comment:
Have you checked the formatting? I am wondering if it needs some indent.
```suggestion
* ``insitu.int`` (`integer`; 0 by default)
    Turns in situ processing on or off and controls how often data is processed.
```
Also, for now, could you put this option in section
```
Diagnostics and output
```
instead of creating a new section
```
Setting up in situ visualization
```
Thanks!!

Reply:
@MaxThevenet Thank you for your suggestions!!!
 You suggest me to put  "Setting up in situ visualization" under "Diagnostics and output". But under "Diagnostics and output", it mentioned three types of diagnostics: FullDiagnostics, BackTransformedDiagnosics and ReducedDiags. you want me to put "Setting up in situ visualization" after above three?

Reply:
Absolutely, at the very end of the `Diagnostics and output` section you could add a sub-section
```rst

In-situ visualization
^^^^^^^^^^^^^^^^^^^^^

Besides the diagnostics described above, WarpX has in-situ visualization capabilities.
This is controlled by the following option(s):

* ``insitu.int`` (`integer`; 0 by default)
    Turns in situ processing on or off and controls how often data is processed.


```

Reply:
@MaxThevenet I'm not familiar with this PR system, Here are my questions: 1) By modifying the file in my warpx fork lge0303/warpx,  I found the version of my fork warpx is too old, do I need to merge it first, then modify the change and commit? 2). after that I need to pull a new requirement or just keep this one and made some comments here?
 Thanks!

Reply:
1. you shouldn't have to do a merge, all you need to do is (i) implement the modifications on your branch `patch-1`, commit the changes and push it, just as you did to create this PR. 
2. You shouldn't have to pull anything, just make the changes we discussed and push on your branch on your fork :)

See our documentation at https://warpx.readthedocs.io/en/latest/developers/contributing.html for more info.